src/index/entry.rs

```diff
@@ -38,30 +38,48 @@ pub struct RuneEntry {
   pub number: u64,
   pub premine: u128,
   pub spaced_rune: SpacedRune,
-  pub supply: u128,
   pub symbol: Option<char>,
   pub timestamp: u32,
 }
 
 impl RuneEntry {
   pub fn mintable(&self, block_height: Height, block_time: u32) -> Result<u128, MintError> {
     let Some(mint) = self.mint else {
-      return Err(MintError::Unmintable(self.spaced_rune.rune));
+      return Err(MintError::Unmintable);
     };
 
     if let Some(end) = mint.end {
       if block_height.0 >= end {
-        return Err(MintError::End((self.spaced_rune.rune, end)));
+        return Err(MintError::End(end));
       }
     }
 
     if let Some(deadline) = mint.deadline {
       if block_time >= deadline {
-        return Err(MintError::Deadline((self.spaced_rune.rune, deadline)));
+        return Err(MintError::Deadline(deadline));
       }
     }
 
-    Ok(mint.limit.unwrap_or(runes::MAX_LIMIT))
+    if let Some(count) = mint.count {
+      if self.mints > count {
```

Comment:
@casey I believe this should be self.mints >= count, will allow 1 mint over count otherwise

Reply:
Good catch! Fixed.